frontend/src/main.ts

```diff
@@ -5,15 +5,18 @@ import './plugins/bootstrap-vue';
 import App from './App.vue';
 import { createRouter } from './router';
 import store from './store';
-import BootsrapVue from 'bootstrap-vue';
-import VueKuzzle from 'vue-kuzzle';
+import { BootstrapVue, BootstrapVueIcons } from 'bootstrap-vue';
```

Comment:
Why do we need BootstrapVueIcons if we are using fontawesome icons likes `<i class="fa fa-question fa-2x"></i>` ?